src/main/controllers/test_connection.ts

```
@@ -1,20 +1,30 @@
 import { NextFunction, Request, Response } from "express";
 import { Kafka } from "kafkajs";
-import _ from "lodash";
+import * as _ from "lodash";
```

ESLint

[error] Replace `"lodash"` with `'lodash'`

(prettier/prettier)